test/AddAndRemoveViewconfTests.js

```diff
@@ -34,8 +34,6 @@ describe('Simple HiGlassComponent', () => {
 
       api = viewer(div, simpleCenterViewConfig, {});
 
-      const p = api.setViewConfig(simpleCenterViewConfig);
```

Comment:
Are you removing this because the return value isn't used? 

It might be there just to be called and ensure no errors are raised.

Reply:
Sorry: That was too aggressive. I'll put it back, but just won't assign it to anything.